### PR TITLE
DCOS-21235: [Firefox] Form control icons overflow issue

### PR DESCRIPTION
**Pull request by GeorgiSTodorov**
Add width to the input to fix overflow after text is entered in Firefox.

Closes DCOS-21235

## Testing
In Firefox, in `Components` tab type something in the search bar. The magnifying glass and the cancel icon shouldn't move when something is typed (see screenshots).

Also, if possible, test this change in other browsers as well - I tested in Chrome to make sure it still works correctly.

## Trade-offs
I spend some time looking and wondering where to put my css change. If someone has better place in mind, please tell me.

## Dependencies
None.

## Screenshots
screenshot from 2018-08-17 13-28-36
screenshot from 2018-08-17 13-30-27
screenshot from 2018-08-17 13-30-53

### CODE REVIEW

**Comment by Fabs** [comment]
I am not sure about the place of where to fix it. Shouldn't it have been fixed on the canvas repo? (https://github.com/mesosphere/cnvs).

@Poltergeist and @mperrotti ?

File: src/styles/components/form-elements/form-group/styles.less [hunk 1]
```diff
@@ -36,6 +36,10 @@
       }
     }
   }
+
+  .health-tab input {
+    width: 70%;
```
[the remark this answers is not in the document]
Comment (GeorgiSTodorov) [reply src/styles/components/form-elements/form-group/styles.less]: Good idea, I've updated the PR.

screenshot from 2018-08-20 10-09-28

**Comment by Fabs** [comment]
I do not know @mperrotti, I think @Poltergeist or @weblancaster would be a better input here.

**Comment by GeorgiSTodorov** [comment]
@Fabs, @weblancaster thumbs-uped the solution that Mike suggested.

**Comment by nLight** [comment]
@mperrotti I believe CNVS is being used to assemble Mesosphere's web-site, documentation and stuff. Design team used to maintain it. As you might have seen in our code base we were overriding it a lot to make it work for DCOS UI. So answering your question, the UI team has never actually maintained CNVS and its current status is unknown to me. @leemunroe any insights here?

**Comment by leemunroe** [comment]
Going forward both the DC/OS UI and Website are not planning to use CNVS directly. So for now probably best to fix it here in this repo and not in CNVS, then make sure we include the fix when we create a UI Kit component.

**Comment by tnguyen0** [comment]
@weblancaster I disagree it is out of scope.

The ticket states:
"In Firefox, the icons for search and clear spill outside the bounds of the .form-control."

And then an example was given. It states it is broken for all instances of `.form-control`.

**Comment by GeorgiSTodorov** [comment]
@tnguyen0 I think it was written a bit ambiguous, which is why nobody noticed. Anyway I created a new jira for fixing the other instances of this bug.

**Pull request merged**